docs/source/installation.rst

```diff
@@ -26,47 +27,30 @@ Instead, use a network firewall to whitelist the server to authorized clients,
 or use a web server configuration directive (such as Apache's 
 ``<LimitExcept GET>``) to set up who is allowed to POST/PUT/DELETE events.
 
-Before you begin
-================
-
-The instructions below assume that you have a Django project already created.
-If you don't, you can create one using the "django-admin.py startproject" 
-command (in an environment where Django has been installed).  The specifics
-of where and how you create your project will depend on your specific hosting
-preferences and needs.  Some understanding of WSGI hosting and Python virtual
-environments is highly recommended before going forward.
-
 Install
 =======
 
-To install this package, do **one (1)** of the following:
+1. Install the package. ::
 
-- Place the ``premis_event_service`` folder in your Django project directory
-  (alongside the ``manage.py`` file)
-- Place the ``premis_event_service`` folder somewhere in your PYTHON_PATH
-- Install using: pip install <path to django-premis-event-service>
-  (activate your Django virtualenv before running)
-.. - Install from PyPI using: pip install django-premis-event-service
-..   (activate your Django virtualenv before running) (NOT YET AVAILABLE)
+    $ pip install django-premis-event-service
 
-Quick start
-===========
 
-1. Follow the Mandatory Configuration instructions in :doc:`configuration`.
+2. Add ``premis_event_service`` to your ``INSTALLED_APPS``. Be sure to add ``django.contrib.admin`` if it is not already present. ::
 
-2. Include the event service URLconf in your project ``urls.py`` like this (do 
-   not modify this line -- this app must be served from top-level URLs for 
-   in its current state)::
+    INSTALLED_APPS = (
+        'django.contrib.admin',
+        # ...
+        'premis_event_service',
+    )
 
-    url(r'', include('premis_event_service.urls')),
+4. Include the URLs. ::
 
-3. Run ``python manage.py syncdb`` to create the event service models. If 
-   asked to create a superuser account, follow the prompts to do so. You will 
-   need at least one superuser account.
+    urlpatterns = [
```

Comment:
Since this doc directs users to add `django.contrib.admin` to `INSTALLED_APPS`, should we indicate to include the admin url pattern `url(r'^admin/', include(admin.site.urls))`?